### ISSUE TITLE
Create a site with the documentation of the package

### ISSUE THREAD

**Issue by FrancoisGuillem** [rte-antares-rpackage/manipulateWidget#18]

**Comment by cderv** [comment]
A first version seems to work. Currently on [cderv/manipulateWidget@docsite](https://github.com/cderv/manipulateWidget/tree/docsite) (could make a PR to work on it if you prefer) However, several issues need to be delt with:

- [ ] CRAN badge is not working when compiling website due to a pandoc error. **See why**
- [ ] Readme.Rmd need to be modified with option `always_allow_html = TRUE` in order to include widget. **need confirmation it is ok to change that**
- [ ] NEWS must be change in NEWS.md with some change in heading in order for `pkgdown` to parse the file correctly. **Need confirmation that it is ok to change**
- [ ] Currently, examples cannot be run as they are widget. **See if we have a solution to do otherwise**
- [ ] Readme and vignette are taken "as is". However, they seems to have the same content. See if we split them up, or if we want to differentiate *github readme* from *Home page in the website*. **It is possible, we just have to decide to do so.**

**Comment by FrancoisGuillem** [comment]
1- This is strange. Have you tried to update your packages?
2- You may need package `webshot` to convert htmlwidgets as image files. 
3- OK for NEWS.md
4- webshot? gifs?
5- I agree. README should be shorter and have a link to the package vignette.

**Comment by cderv** [comment]
Yes I was talking about examples in "reference" section. They are not run because they produce a widget. I consider it is ok not to run them as you give examples of usage in vignettes. 

Readme should be shorter we agree and it sould have a link to website when it will be ready, rather than the vignette. We could discuss Readme content in another issue if you want.

**Comment by FrancoisGuillem** [comment]
Je suis d'accord pour les exemples, c'est pas grave si on peut pas les exécuter. Et pour tout le reste, je suis OK. Quand tu as fini de corriger les petits soucis, tu peux créer une PR avec le README et la vignette actuels ? Je les ai pas mal modifiés sur ma branche, donc je m'occuperai de les découper correctement.

**Comment by FrancoisGuillem** [comment]
Pour le badge, c'est pas un soucis de proxy ? Quand j'essaie de générer le fichier dans Rstudio, il essaie de télécharger le badge et de le base64-encoder mais il se fait envoyer bouler par le proxy !

**Comment by cderv** [comment]
Proxy peut être, j'essaierai de nouveau dès que possible. En fait, J'ai une erreur côté pandoc, pour la conversion en html donc. 

Pour réintégrer le travail, je peux repasser sur une de tes branches sinon afin de faire un rebase de la mienne et vérifier que ça marche. 
Sinon je ferais ce que tu me demandes.

**Comment by FrancoisGuillem** [comment]
C'est comme tu le sens. C'est la branche develop.

**Comment by cderv** [comment]
Avancement : 
* J'ai intégré ta nouvelle vignette et ton nouveau readme pour générer un nouveau site. 
* J'ai également proposé un template [bootswatch](https://bootswatch.com/) pour mettre un peu de couleur. A voir si vous préférez un template perso ou un des autres templates bootswatch dispo.
* J'ai proposé une petite modif dans le readme pour passer le _title_ en _titre1_ afin que la page d'accueil ait bien le titre. 
* Je n'arrive toujours pas à inclure le badge à cause d'une erreur pandoc, et j'ai essayé de chez moi où je n'ai pas de proxy. Donc ça ne doit pas être ça. 

Il me reste donc encore à comprendre pourquoi ça marche pas. Dans le but d'avoir une première version, je propose entre temps de créer une page d'accueil du site dédiée indépendante du readme. De cette façon, pas de doublons entre la page d’accueil et la vignette. (ce qui est tjs le cas actuellement Pour cela, il me reste à compléter le readme comme demandé en #19, faire les ajustements et je prépare une PR propre pour intégration.

**Comment by FrancoisGuillem** [comment]
Ca y est j'ai découpé la vignette et le README. Ils sont disponible dans la branche develop

**Comment by cderv** [comment]
Impec. Dans mes derniers tests, j'ai toujours besoin dans le README 

* de commenter le bagde CRAN car pas fonctionnel avec lors de la conversion en html avec pandoc (tjs pas trouvé la solution)
* de rajouter `always_allow_html = TRUE` pour compiler le rmarkdown en html

Pour le reste ça fonctionne bien en compilant le site avec `pkgdown::build_site(example = F)`. J'ai fait un fichier de config `_pkgdown.yml` pour tuner un peu le site. 

Faut que je reparte de ton dernier commit de develop pour recréer une branche ç jour- mais tu peux tester direct chez toi si tu veux entre temps.

**Comment by cderv** [comment]
Je continue le boulot là dessus. J'ai plus de problème avec les badges, ça fonctionne maintenant. 
Par contre, comme votre `readme.Rmd` et `readme.md` contient des *gif* aussi utile dans la vignette, ça pose problème car ce n'est pas vraiment compatible par défaut. Je cherche comment faire.
Je vais pousser le travail en PR même si ce n'est pas fini pour que ça soit plus simple pour vous de voir à quoi ça ressemble. C'est dans une de mes branches sur le fork pour le moment.
